src/js/components/search/filters/timePeriod/FiscalYear.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * FiscalYear.jsx
+ * Created by Emily Gullo 11/03/2016
+ **/
+
+import React from 'react';
+
+const propTypes = {
+    name: React.PropTypes.string
+};
+
+export default class FiscalYear extends React.Component {
+
+    render() {
+        return (
+            <div className="fiscalYearOption">
+                <input type="checkbox" id={this.props.name} value={this.props.name} />
```

Comment:
Again, let's make the HTML IDs valid (no spaces). Maybe just have the parent provide the year and prepend the "FY" string in this component's `label` and them building a compliant HTML ID with that.

Review ES6's [template strings feature](https://developers.google.com/web/updates/2015/01/ES6-Template-Strings#string_substitution).